src/Gui/BitmapFactory.cpp

```diff
@@ -304,21 +311,15 @@ QPixmap BitmapFactoryInst::pixmapFromSvg(const char* name, const QSizeF& size,
         QFile file(iconPath);
         if (file.open(QFile::ReadOnly | QFile::Text)) {
             QByteArray content = file.readAll();
-            icon = pixmapFromSvg(content, size, colorMapping);
+            icon = pixmapFromSvg(content, QSizeF{size.width() * dpr, size.height() * dpr}, colorMapping);

```

Comment:
IIRC QSize does implement `operator*` with qreal: https://doc.qt.io/qt-6/qsize.html#operator-2a-1 so it should be as simple as:
```suggestion
            icon = pixmapFromSvg(content, size * dpr, colorMapping);
```

Wouldn't it work?

Reply:
Yes it works. I wasn't aware it. Thanks.